pulsar-broker/src/test/java/org/apache/pulsar/broker/web/WebServiceTest.java

```diff
@@ -387,7 +417,7 @@ private String makeHttpRequest(boolean useTls, boolean useAuth) throws Exception
     }
 
     private void setupEnv(boolean enableFilter, boolean enableTls, boolean enableAuth, boolean allowInsecure,
-                          double rateLimit, boolean disableTrace) throws Exception {
+                          double rateLimit, boolean disableTrace, boolean enableCompressionSupport) throws Exception {
```

Comment:
I know you just added a flag, but honestly, how can someone reading the setupEnv(...`false`) really know you enabled compressionSupport?

I would either:
1. Write a `EnvBuilder` , that has `enableCompression()` 
2. Add /* enableCompression= */ to help people who read this code on the lines calling it.

BTW: Builder would not force you to change so many lines when adding a method, since they all have defaults.

Reply:
>Add a new parameter to control whether to enable the Serverlet response compression support or not. Even though enabling the response compression is based on the client request header Accept-Encoding: gzip, it is a breaking change for current Pulsar versions, and if this feature can't be disabled, the user will be confused if they upgraded Pulsar to a new version but their Prometheus server is still in an old version which can't disable the compression request.

Sorry, I do not understand why removed the flag. Is the above issue resolved?